homeassistant/components/fortigate/__init__.py

```diff
@@ -0,0 +1,67 @@
+"""Fortigate integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_USERNAME, CONF_PASSWORD, \
+    CONF_DEVICES, EVENT_HOMEASSISTANT_STOP
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.discovery import async_load_platform
+
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'fortigate'
+
+DATA_FGT = DOMAIN
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_DEVICES, default=[]):
+            vol.All(cv.ensure_list, [cv.string]),
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+
+async def async_setup(hass, config):
+    """Start the fortigate component."""
+    conf = config[DOMAIN]
+
+    host = conf.get(CONF_HOST)
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values. We only need to use `dict.get` if we don't know that the key is in the dict.

Reply:
fixed